src/style.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  border-radius: 0.5px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--5 > .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 > .stars__star:first-child {
```

combine selectors with comma
```suggestion
.stars--5 > .stars__star,
.stars--1 > .stars__star:first-child {
  background-image: url(images/star-active.svg);
}
```